ivy_payment_gateway.php

```diff
@@ -0,0 +1,107 @@
+<?php
+global $requestUrl;
+class Ivy_Pay extends WC_Payment_Gateway
+{
+  function __construct()
+  {
+    // global ID
+    $this->id = "ivy_payment";
+    // Show Title
+    $this->method_title = __("ivy_Pay", 'ivy_pay');
```

Comment:
```suggestion
    $this->method_title = __("ivy_Pay", 'ivy_pay');
```
@YousafEsparks can this title be changed here?